libs/vkd3d/device.c

```diff
@@ -610,6 +610,8 @@ static const struct vkd3d_instance_application_meta application_override[] = {
      * Use no-staggered-submit by default on UE. We've only observed issues in Wukong here, but
      * unless we see proof that UE titles want staggered,
      * we'll disable for now to be defensive and de-risk any large scale regressions. */
+    /* World of Warcraft retail. Broken MSAA code where it renders to multi-sampled target with single sampled PSO. */
+    { VKD3D_STRING_COMPARE_EXACT, "Wow.exe", VKD3D_CONFIG_FLAG_FORCE_DYNAMIC_MSAA, 0 },
```

Comment:
should probably be moved above the UE5 comment to avoid confusion

Reply:
Derp. Fixed.